Packs/Core/ReleaseNotes/3_0_68.md

```diff
@@ -0,0 +1,10 @@
+
+#### Playbooks
+
+##### Large Upload Alert
+
+- Added transformers to some conditions within the *'Calculate Verdict'* task to obtain the highest or lowest score in cases where an array of results was returned.
+- Fixed an issue which caused the *'Calculate Verdict'* task to fail due to an incorrect filter being used in certain conditions of the *'Calculate Verdict'* playbook task.
+- Fixed the context path for risk scores of risky hosts and users within the *'malicious'* condition of the *'Calculate Verdict'* task.
+- Updated task numbers 32 and 48's descriptions and names.
```

Comment:
Didn't see that on your PR - maybe the format reverted it?

Reply:
![Screenshot 2024-09-24 at 17 04 19](https://github.com/user-attachments/assets/95575db4-dcd4-416e-8653-874cc1a1991c)
![Screenshot 2024-09-24 at 17 04 50](https://github.com/user-attachments/assets/c8e12ccb-6ed5-41cb-8da3-d5d6d336ada1)
![Screenshot 2024-09-24 at 17 05 31](https://github.com/user-attachments/assets/6b023d46-7892-49ce-9504-01ef604493f2)
here are the changes and I committed another change to task 48 name